forms/snippets/add_item.py

```diff
@@ -21,15 +21,15 @@
 
 SCOPES = "https://www.googleapis.com/auth/forms.body"
 API_KEY = "<YOUR_API_KEY>"
-DISCOVERY_DOC = f"https://forms.googleapis.com/$discovery/rest?version=v1beta&key={API_KEY}&labels=FORMS_BETA_TESTERS"
+DISCOVERY_DOC = f"https://forms.googleapis.com/$discovery/rest?version=v1&key={API_KEY}"
```

Comment:
While not harmful, shouldn't be necessary to include this once GA and the API client is updated. Specifying the api name and version is sufficient.

Reply:
OK, fixed and tested—it's working w/o the key now.